proc-macro/src/lib.rs

```diff
@@ -30,6 +30,54 @@ use synstructure::{
     Structure,
 };
 
+/// Register type sizes for [EventsDecoder](struct.EventsDecoder.html)
```

Comment:
I think we should also mention the module name constant, which will be the same as the trait ident. This in turn must match the name of the substrate pallet. Indeed the codegen for events and calls depends on this constant, so we might mention that this attribute is required